README.md

```diff
@@ -3,7 +3,7 @@
 To start your Phoenix app:
 
   * Install dependencies with `mix deps.get`
-  * Start Phoenix endpoint with `mix phoenix.server`
+  * Start Phoenix endpoint with `PORT=4000 mix phoenix.server`
```

Comment:
Looks like the issue is actually coming from `dev.exs`.

https://github.com/Dkendal/snek/blob/master/config/dev.exs#L10

Ideally, that should be `http: [port: 4000]`.

Just doing some casual :trollface:'ing